src/grpc/hpr_packet_up.erl

```diff
@@ -60,10 +60,10 @@ rssi(Packet) ->
 frequency(Packet) ->
     Packet#packet_router_packet_up_v1_pb.frequency.
 
--spec frequency_mhz(Packet :: packet()) ->
-    float() | integer() | infinity | '-infinity' | nan | undefined.
+-spec frequency_mhz(Packet :: packet()) -> float().
 frequency_mhz(Packet) ->
-    Packet#packet_router_packet_up_v1_pb.frequency / 1_000_000.
+    Mhz = Packet#packet_router_packet_up_v1_pb.frequency / 1000000,
+    list_to_float(float_to_list(Mhz, [{decimals, 4}, compact])).
```

Comment:
Could also do `round(Mhz * 10_000)/10_000`.

Reply:
Not quite the same.
```
1> Hz = 865_062_502.                                                                 
865062502

2> Mhz = Hz / 1_000_000.                                                              
865.062502

3> (Mhz * 10_000) / 10_000.
865.062502

4> list_to_float(float_to_list(Mhz, [{decimals, 4}, compact])).
865.0625
```

Reply:
You left out `round`:
```8> Hz = 865_062_502.                                                            865062502
9> Mhz = Hz / 1_000_000.                                                        865.062502
10> round((Mhz * 10_000)) / 10_000.
865.0625
11> 
```

Reply:
I absolutely did.